<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsDialog</class>
 <widget class="QDialog" name="settingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZIMA-PTC-Cleaner Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../zima-ptc-cleaner.qrc">
   <normaloff>:/gfx/icon.png</normaloff>:/gfx/icon.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="enableSystemContextMenu">
     <property name="text">
      <string>Enable system context menu</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkOldFiles">
     <property name="text">
      <string>Remove old files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkFilters">
     <property name="text">
      <string>Remove all files matching these filters:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listFilter">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Mask:</string>
       </property>
       <property name="buddy">
        <cstring>textFilter</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textFilter"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemove">
       <property name="text">
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
 <include location="../zima-ptc-cleaner.qrc"/>
 </resources>
 <connections/>
</ui>
